src/kontrol/utils.py

```diff
@@ -112,3 +104,40 @@ def kontrol_file_contents() -> str:
 $ kontrol command --help
 ```
 """
+
+
+def kontrol_toml_file_contents() -> str:
+    return """[build.default]
+foundry-project-root       = '.'
+regen                      = true
+rekompile                  = true
+verbose                    = true
+debug                      = false
+require                    = 'lemmas.k'
+module-import              = 'TestBase:KONTROL-LEMMAS'
+
+[prove.default]
+foundry-project-root       = '.'
+verbose                    = true
+debug                      = false
+max-depth                  = 25000
+reinit                     = false
```

Comment:
Just to say that we have a little bit of an issue with `--reinit`, in that (as far as I know) `setUp` is not re-run unless we say `--reinit` explicitly even if the code has changed. I don't think this affects its default being `false`.